src/index.html

```diff
@@ -7,7 +7,7 @@
     <title>Document</title>
 </head>
 <body>
-    <main class="content"> 
```

Comment:
>## I see you refactored this section of the code 👍🏻

Reply:
Yes, I had to do that, because it was causing conflicts of spacing nature with the footer. But later on, or even now that you are going to do the "home" kanban section, you can modify it again.